### PR TITLE
Player Crouch Animation completed

### PR DESCRIPTION
**Pull request by KathiravanThirumurthy**
Player Crouch Animation completed

### CODE REVIEW

**Comment by KathiravanThirumurthy** [comment]
Player Crouch Animation completed

File: Assets/Scripts/PlayerAnimation.cs [hunk 1]
```diff
@@ -2,12 +2,17 @@
 using System.Collections.Generic;
 using UnityEngine;
```
Comment (ahujarahul99) [inline Assets/Scripts/PlayerAnimation.cs:4]: We need namespaces also .

File: Assets/Scripts/PlayerAnimation.cs [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
 {
     private Animator anim;
     private void Awake()
     {
-        anim = GetComponentInChildren<Animator>();
+        //anim = GetComponentInChildren<Animator>();
+        anim = GetComponent<Animator>();
     }
     // Start is called before the first frame update
     void Start()
```
Comment (ahujarahul99) [inline Assets/Scripts/PlayerAnimation.cs:18]: Please do not write empty start update method here .

File: Assets/Scripts/PlayerController.cs [hunk 3]
```diff
@@ -1,62 +1,107 @@
...
+            // setting the value to false to confirm Control is not pressed
+            isCrouch = false;
+            // calling the crouching method from the PlayerAnimation Script by passing the current value
+            _playerAnimation.crouching(isCrouch);
+        }
+        // to check playermovement is possible only when the Crouch animation is not playing
+        if(!isCrouch)
+        {
+            // moving the player using velocity with speed
+            rgd.velocity = new Vector2(move * _speed, rgd.velocity.y);
```
Comment (ahujarahul99) [inline Assets/Scripts/PlayerController.cs:89]: Rigidbody work should be done in the fixed update .

**Pull request merged**